main/end-to-end-tests/cypress/support/commands.js

```diff
@@ -51,6 +51,12 @@ Cypress.Commands.add('login', role => {
       email: Cypress.env('researcherEmail'),
       password: Cypress.env('researcherPassword'),
     };
+  }
+  if (role === 'restrictedResearcher') {
+    loginInfo = {
+      email: Cypress.env('restrictedResearcherEmail'),
```

Comment:
Remember to add this to GH Actions. Also add the creds [here](https://quip-amazon.com/4gSRAngBjLcO/Trigger-E2E-Test-GitHub-Action-Locally#QTb9CArMy7D)

Reply:
Yes, it's on my list. I wanted to wait on doing that till we merge the feature into develop, similar to the other e2e tests for appstream.